kubernetes/ansible/roles/kong-api/tasks/main.yml

```diff
@@ -12,9 +12,5 @@
 - name: Copy kong api scripts
   copy: src=static-files/kong-api-scripts dest=/tmp mode=0755
 
```

Comment:
We've to have a check whether that port is accessible or not.

Reply:
this task file is no more used for kubernetes in release-2.10.0 @rjshrjndrn , it is a different role altogether kubernetes/ansible/roles/kong-api/

Reply:
then which role is used to onboard APIs ? ansible/role ?. I think it might be good to add wait_for to check the port in the file, because most probably, we're going to use this file in future.

Reply:
kubernetes/ansible/roles/kong-api/tasks/main.yml is the file which is used and this defaults/main.yml is being symblinked

Reply:
kubernetes/ansible/roles/kong-api/tasks/main.yml is the role yml which will be used and kubernetes/ansible/api-manager.yml is the yml being used and ansible/roles/kong-api/defaults/main.yml is symblinked.